internal/service/project/resource_project.go

```diff
@@ -164,8 +164,22 @@ func (r *projectRS) Create(ctx context.Context, req resource.CreateRequest, resp
 		return
 	}
 
+	projectPropsParams := &PropsParams{
+		Context:      ctx,
+		IsDataSource: false,
+		ProjectID:    projectID,
+		Warnings:     &resp.Diagnostics,
+	}
+
+	projectApis := &APIClients{
+		ProjectsAPI:           connV2.ProjectsApi,
+		TeamsAPI:              connV2.TeamsApi,
+		PerformanceAdvisorAPI: connV2.PerformanceAdvisorApi,
+		MongoDBCloudUsersAPI:  connV2.MongoDBCloudUsersApi,
+	}
+
```

Comment:
```suggestion
	projectPropsParams := &APIClients{
	        ProjectID:    projectID,
	        IsDataSource: false,
		ProjectsAPI:           connV2.ProjectsApi,
		TeamsAPI:              connV2.TeamsApi,
		PerformanceAdvisorAPI: connV2.PerformanceAdvisorApi,
		MongoDBCloudUsersAPI:  connV2.MongoDBCloudUsersApi,
	}

```

And keep (ctx, projectPropsParams, &resp.Diagnostics) as the arguments?